core/modules/system/config/system.core.json

```
@@ -80,6 +80,13 @@
     "user_mail_status_canceled_notify": 0,
     "user_register": "admin_only",
     "user_signatures": 0,
+    "user_password_constraints": {
+        "constraints_active": "1",
+        "minlength": "8",
+        "not_username": 1,
+        "not_email": 1,
+        "no_case": 0
```

No strong feelings about this, but we could be more descriptive for better DX:

```suggestion
    "user_password_constraints": {
        "constraints_active": "1",
        "min_length": "8",
        "not_same_as_username": 1,
        "not_same_as_email": 1,
        "case_insensitive_comparison": 0
```